src/hooks/useCurrentWeather.js

```diff
@@ -0,0 +1,35 @@
+import { useEffect } from "react";
+import fetchWeather from "../api/fetchWeather";
+
+const useCurrentWeather = (setWeatherData, latitude, longitude, initData) => {
+  const handleError = (error) => {
```

Comment:
function just for function, you don't need handle error use setWeatherData in place